### ISSUE TITLE
Edit menu item titles

### ISSUE THREAD

**Issue by pkamb** [knurling/ServiceStation#5]
Currently, Service Station menu items have titles equivalent to their underlying application or script.

Applications use the app's `CFBundleDisplayName` if available. Scripts use the script's filename minus the path extension.

**Comment by luckman212** [comment]
This is a must-have feature! (please)

**Comment by gordon8214** [comment]
Yes please. I typically want the menu item title to reflect the thing that will happen when an app is used with a particular file type. For example "Open folder in Terminal" (Terminal) or "Uninstall" (AppDelete).

**Comment by weavermedia** [comment]
Yes please! Loving the app so far and small touches like this would really make it shine! 💎

**Comment by stvkmco** [comment]
+1
